sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -346,7 +358,7 @@ class ProjectOverviewView extends VerticalLayout{
     }
 
     private void showWhenFailingSamplesExist(Component component) {
-        component.setVisible(failingSamplesExist())
+        component.setVisible(false)
```

Comment:
This should not change. Always setting it to false is dangerous here since you are introducing the assumption that this method is created in the beginning and thus limiting the time point where a component can be set up by it.
As you want to have the component visible whenever failing samples exist for a selected project, you have to make sure this is the case regardless on when you call the method.

Reply:
Good catch, I did not intent to change that. I changed it back.